JPS_Ontology/ontology/ontoems/OntoEnvironmentalMeasurementStation.csv

```diff
@@ -11,6 +11,8 @@ WaterLevel,Class,http://purl.org/iot/vocab/m3-lite#WaterLevel,IS-A,,,,"The heigh
 Rainfall,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Height,IS-A,,,,The depth of rain that accumulated over a measurement time quantity.,https://www.theworldavatar.com/kg/ontoems/,Rainfall
 Rainfall,Class,https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/WeatherOntology.owl#Precipitation,IS-A,,,,The depth of rain that accumulated over a measurement time quantity.,https://www.theworldavatar.com/kg/ontoems/,Rainfall
 Rainfall,Class,http://purl.org/iot/vocab/m3-lite#Rainfall,IS-A,,,,The depth of rain that accumulated over a measurement time quantity.,https://www.theworldavatar.com/kg/ontoems/,Rainfall
+SnowFall,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Height,IS-A,,,,The depth of snow that accumulated over a measurement time quantity.,https://www.theworldavatar.com/kg/ontoems/,Snowfall
```

Comment:
My mistake, I gave the wrong definition. The definition of the snowfall parameter from Open-Meteo API (https://open-meteo.com/) seems to be incremental measurement: "Snowfall amount of the preceding hour in centimeters.".

Reply:
> My mistake, I gave the wrong definition. The definition of the snowfall parameter from Open-Meteo API (https://open-meteo.com/) seems to be incremental measurement: "Snowfall amount of the preceding hour in centimeters.".

Then I believe this should be the description of this concept, instead of this vauge "over a mearuement time quantity". Otherwise tricky to guess that duration, but happy to discuss

Reply:
Discussed and decided to keep the description general.